physics/pert-eff-mass.cpp

```diff
@@ -0,0 +1,123 @@
+#include <LatCore/OptParser.hpp>
```

Comment:
this file is too specific to go into LatAnalyze, please move it to your analysis code